### PR TITLE
feat(flex): FRMW-118-Document static flex items

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-118

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM 

---
- Document and provide an example of static flex items in `EncoreUI` layout.  
- Reformat to conform to HTML styleguide

### CODE REVIEW

**Comment by 100stacks** [comment]
screen shot 2015-11-03 at 10 23 21 am

**Comment by 100stacks** [comment]
screen shot 2015-11-03 at 10 31 42 am

**Comment by bry4n** [comment]
LGTM

File: src/layout/docs/responsiveLayout.html [hunk 1]
```diff
@@ -78,12 +85,44 @@
...
 
-    <p>See the <a href="#/components/layout#Options">layout options</a> for more information on responsive flex attributes.</p>
+    <p>See the <a href="#/components/layout#Options">layout options</a> for more information on responsive flex 
+    attributes.</p>
 
+    <h3>Static Flex Options</h3>
+
+    <div layout="row">
+      <div flex class="flex-fix flex-box dark-blue">
+        [flex="0 0 200px"]
```
Comment (CITguy) [inline src/layout/docs/responsiveLayout.html]: Please change content to reflect the CSS property, not the HTML attribute.

```HTML
flex: 0 0 200px;
```

**Comment by 100stacks** [comment]
@CITguy okay change made :+1:

File: src/layout/docs/responsiveLayout.html [hunk 2]
```diff
@@ -99,9 +138,15 @@
...
+    <p>Note that the <code>flex-order</code> attribute is not compatible with the <code>layout-margin</code> attribute. 
+    This is because the CSS selector engine selects based on DOM markup order and the <code>layout-margin</code> 
+    attribute makes use of <code>:first-child</code> and <code>:last-child</code> to apply margins to only the inner 
+    elements in the container. As a work-around, <code>[flex-order].left-margin</code> and 
+    <code>[flex-order].right-margin</code> classes are availabe to manually add margins. The most likely use cases for 
+    these classes is for programatic ordering of children.</p>
```
Comment (glynnis) [inline src/layout/docs/responsiveLayout.html]: "programatic" --> programmatic (two Ms)

**Comment by glynnis** [comment]
After spelling is corrected, LGTM

**Comment by CITguy** [comment]
@100stacks, please rebase.

**Comment by 100stacks** [comment]
@glynnis comments implemented....@CITguy rebased PR :+1:

File: src/layout/docs/responsiveLayout.html [hunk 3]
```diff
@@ -78,12 +85,44 @@
...
 
-    <p>See the <a href="#/components/layout#Options">layout options</a> for more information on responsive flex attributes.</p>
+    <p>See the <a href="#/components/layout#Options">layout options</a> for more information on responsive flex 
+    attributes.</p>
 
+    <h3>Static Flex Options</h3>
+
+    <div layout="row">
+      <div flex class="flex-fix flex-box dark-blue">
+        flex=0 0 200px;
```
Comment (CITguy) [inline src/layout/docs/responsiveLayout.html]: Please change the `=` to a colon (`:`) to match CSS property definition.

```
flex: 0 0 200px;
```

**Comment by 100stacks** [comment]
@CITguy updated with comments :+1:

File: src/layout/docs/responsiveLayout.html (hunk 3 above)
Comment (bry4n) [inline src/layout/docs/responsiveLayout.html]: nice catch

**Comment by ssalinas24** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**